source/assets/public/index.js

```diff
@@ -1,10 +1,10 @@
 /**
  * JavaScript goes here
  */
-
 function mobileMenu() {
+  var NAV_ACTIVE_CLASS = 'js-active';
   var body = document.body;
-  var mobileNavContainer = document.querySelector('.nav--mobile');
+  var mobileNavContainer = document.querySelector('nav');
```

Comment:
I do think we should be more specific about this selector, eg the footer nav could also legitimately be a `nav` element - so maybe putting in a `js-nav` class or something like that

Reply:
Can do - it seemed a bit redundant to have a `<nav class="nav">`.
Are you OK with ID selectors? We could just use `#menu` here.

Reply:
Sure, an ID selector makes sense in this context